src/pages/SelectLocationsPage.js

```diff
@@ -153,7 +153,10 @@ class SelectlocationsPage extends React.Component {
           this.selectProvinceError.focus()
         }
       })
-      .catch(err => console.log(err))
+      .catch(err => {
+        console.log(err)
```

Comment:
Please use the './utils/logger.js' logError class? 

As it stands right now this will be a linting error, and we should be logging errors to stderr or console.error(...arguments)

Reply:
Those were there before and I was even thinking of removing them since they print to browser console, don't think they're useful. 

I'll check with the team to see if there's any objections in removing them.